tests/client/feedback/training/test_schemas.py

```diff
@@ -43,6 +45,12 @@
     "required": True,
     "labels": ["1", "2"],
 }
+ranking_question_payload = {
+    "name": "label",
+    "description": "label",
+    "required": True,
+    "values": ["1", "2"],
+}
```

Comment:
IMO we should define all those `rating_question_payload`, `label_question_payload` and `ranking_question_payload` as separate `pytest.fixtures`, and then re-use those when needed